languages/thingtalk/load-thingpedia.js

```diff
@@ -151,6 +152,12 @@ class ThingpediaLoader {
     _addOutParam(functionName, pname, type, typestr, canonical) {
         this._grammar.addRule('out_param_' + typestr, [canonical], this._runtime.simpleCombine(() => new Ast.Value.VarRef(pname)));
 
+        if (!pname.startsWith('address') && !pname.endsWith('Count') && !pname.startsWith('numberOf')) {
+            let postags = posTag(canonical.split(' '));
```

Comment:
No. The POS tag should be in the annotation name already.

Reply:
Fixed in d0c94f3. The reason to have this is we always generate base form even if there is no noun phrase found. But I think it's fine to remove it - it will generate werid sentence but won't generate ambiguity.